aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -108,13 +106,36 @@ http://localhost/Speaker/Evaluations#SpeakerEvaluations
 ```
 Hash tags are useful when doing client side applications. They can be used for easy marking and searching in JavaScript for example.
 
+- - -
+
+### asp-area
+
+```asp-area``` sets the area name that ASP.NET core will use to set the appropriate route with.  Below are examples of how the area attribute causes a remapping of routes.  That is, by setting ```asp-area``` to Blogs will invoke the Areas functionality such that the directory Areas/Blogs will prefix the associated controllers and views for this anchor tag.
 
-## asp-area
+* Project name
 
-```asp-area``` allows for a razor view area to be associated with the hyperlink URL. That is, if you have a project setup in a similar fashion to the project pictured below, the link generated will include as its first segment the area you mention.
+  * wwwroot
 
-![](../_static/ProjectControllersArea.png)
+  * Areas
 
+    * Blogs
+
+      * Controllers
+
+        * HomeController.cs
+
+      * Views
+
+        * Home
+
+          * Index.cshtml
+          
+          * AboutBlog.cshtml
+          
+  * Controllers
+  
+
+        
 Specifying an area tag that is valid, such as ```area="Blogs"``` when referencing the ```AboutBlog.cshtml``` file will look like the following using the ```AnchorTagHelper```.
```

Comment:
Comma after "file".

Again, I'd go with "Anchor Tag Helper" (in plain text) in all of these places where it has to be read.